src/main/java/core/basesyntax/report/convertdata/DataConvertorImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.report.convertdata;
+
+import core.basesyntax.model.FruitOperation;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConvertorImpl implements DataConvertor {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class declaration.